bindings/rust/s2n-tls-hyper/tests/common/echo.rs

```diff
@@ -17,7 +17,7 @@ async fn echo(
     Ok(Response::new(req.into_body().boxed()))
 }
 
-async fn serve_echo<B>(
+pub async fn serve_echo<B>(
```

Comment:
Nit: should this be pub or pub(crate)?

Reply:
I do think it should be `pub`, because `pub(crate)` feels a bit redundant. This is in a submodule of the test folder, so the entire module is already private and not available to external consumers.